config/menus.yml

```diff
@@ -319,14 +319,14 @@ submenu_procedures:
             data-extern-dataport: 'documentsAndMapSettings'
     map_settings:
         label: 'map_settings.menu'
-        path: 'DemosPlan_map_administration_overview'
+        path: 'DemosPlan_element_administration'
         child_paths: [
             'DemosPlan_map_administration_gislayer',
             'DemosPlan_map_administration_gislayer_new',
             'DemosPlan_map_administration_map',
         ]
         permission: ['area_admin_initial_map_view'] # Only controls the menu item. 'area_admin_map' is required for the map settings
-        path_params: ['procedureId']
+        path_params: ['procedure']
```

Comment:
I guess the parameter contains a string? (procedureId)
Therefore I would prefer the name "procedureId" if possible.

Reply:
hey, this url is used in another templates that are not touched in this PR, if I adjust it, I will have to adjust the other templates (which is more a FE thing) I can do it but then it becomes a bit of a bigger scope